aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -44,6 +44,33 @@ static inline Tensor& unary_op_impl_out(Tensor& result, const Tensor& self, Stub
   return result;
 }
 
+// Mimics returning a floating point tensor for complex inputs by default
+// Note: This is done by running the operation as usual and then copying the
+// operation's result to the expected result type.
+template <typename Stub>
+static inline Tensor& unary_op_impl_out_complex_to_float(Tensor& result, const Tensor& self, Stub& stub) {
```

Comment:
maybe rename this to `unary_op_impl_complex_to_float_out` to be more consistent with out current naming conventions

Reply:
Sounds good.